### ISSUE TITLE
Add the current name="***" from the steam profile to the ban list when banning someone with csmm

### ISSUE THREAD

**Issue by Heliox585** [CatalysmsServerManager/7-days-to-die-server-manager#792]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
It would be great if the current profilename of the user that gets banned would be added to the blacklist entry in the serveradmin.xml

    <blacklisted steamID="*" unbandate="*" reason="*" /> 
    <blacklisted steamID="*" name="*" unbandate="*" reason="*" />  

**Describe alternatives you've considered**
Adding the name after banning with csmm or creating a entry directly in the serveradmin.xml are alternatives

**Additional context**
None

**Comment by niekcandaele** [comment]
What is the difference between the blacklist feature from serveradmin.xml and the regular ban list?

Any user that gets banned, and thus ends up on the ban list also automatically gets added to the GBL

**Comment by Heliox585** [comment]
What is the regular ban list? We only have the serveradmin.xml with the bans.

Also its not about the GBL.

Its only about the name of the player in the blacklisted entry, as i have written above.

**Comment by niekcandaele** [comment]
Okay so I had to go ingame and test this to make sure I'm not going crazy :smile: 

The blacklist you mentioned = the regular ban list I mentioned.
I just banned myself from my server and it created an entry in serveradmin.xml



The ban button on CSMM executes the `ban` command on your server, thus it automatically adds it to that blacklist. If that does not happen, then the player does not actually get banned. I tested on a 7d2d server with Allocs fixes and CPM installed.
